src/modules/logger/logger.h

```diff
@@ -58,6 +58,20 @@ namespace px4
 namespace logger
 {
 
+enum class SDLogMode {
+	ARM_UNTIL_DISARM = 0,
+	BOOT_UNTIL_DISARM,
+	BOOT_UNTIL_SHUTDOWN,
+	N_MODES
+};
+
+enum class SDLogProfile {
```

Comment:
This needs to explicitly be `int32_t` (`enum class SDLogProfile : int32_t {`), because it's used as an argument to `param_get`.